tests/testthat/test-use-description.R

```diff
@@ -0,0 +1,81 @@
+context("use_description")
+
+test_that("build_description_list() defaults to values built into usethis", {
+  withr::with_options(
```

Comment:
I think it's easier to read if you use `local_options()`

Reply:
AHA! done.